apps/geocoding-api/build.gradle

```diff
@@ -0,0 +1,129 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+plugins {
+    id 'jacoco'
+    alias libs.plugins.springBoot
+    alias libs.plugins.springDependencies
+}
+group = 'gov.cdc'
+version = '1.0.0-SNAPSHOT'
+
+bootJar {
+    manifest {
+        attributes 'API-Version': archiveVersion
+    }
+}
+
+jar {
+    //  packages the application and spring into one jar
+    enabled = false
+}
+
+dependencies {
```

Comment:
not sure what all this service will touch, but might be able to remove several of these dependencies

Reply:
Removed some